CHANGELOG.md

```diff
@@ -5,42 +5,45 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased][unreleased]
 
 ### Added 
-- Support 10.3 tokens
-- Add "?ping" for easier testing
-- (DotNet) Support Windows authentication
-- Support federated service
+- Support for 10.3 tokens
+- Added "?ping" for easier testing
+- (DotNet) Support for Windows authentication
+- Support for federated services
 
 ### Fixed
-- Better referrer handling
-- Support ? in password
-- Multiple redirects
-- Handle token expiration
-- Remove HTTP range-request Header
+- Better referer handling
+- Support for '?' in passwords
+- Handling of multiple redirects
+- Handling of token expiration
+
+### Removed
```

Comment:
took your comments about range-request headers to heart, laid down another commit and rebased.